sql/moz-fx-data-shared-prod/accounts_backend_derived/users_first_seen_v1/query.sql

```diff
@@ -0,0 +1,41 @@
+WITH users_services_daily_new_entries AS (
+  SELECT DISTINCT
+    user_id_sha256,
+    country,
+    seen_in_tier1_country,
+    registered
+  FROM
+    accounts_backend.users_services_daily
+  WHERE
+  {% if is_init() %}
+    DATE(submission_date) < CURRENT_DATE
+  {% else %}
+    DATE(submission_date) = @submission_date
+  {% endif %}
+),
+existing_entries AS (
+  SELECT
+    user_id_sha256
+  FROM
+    accounts_backend_derived.users_first_seen_v1
+  WHERE
+{% if is_init() %}
+    FALSE
+  {% else %}
+    DATE(submission_date) < @submission_date
+  {% endif %}
+)
+SELECT
+  new_entries.*,
+  DATE(@submission_date) AS first_seen_date,
```

Comment:
This seems to be inconsistent in relation to `users_services_first_seen_v1` which does not seem to have `first_seen_date`, but instead has `submission_date`:
https://github.com/mozilla/bigquery-etl/pull/5210/files#diff-ff00b83b200545f893fc543ef891ac6cf2e9613f36067c0badf461bc87fd60c6R30

I think we should follow the same naming as the only real difference between the two is the grain of the table.